src/server/database/migrations/20180320152455-voteSettings.js

```diff
@@ -0,0 +1,43 @@
+import {RETROSPECTIVE} from 'universal/utils/constants';
+
+exports.up = async (r) => {
+  try {
+    await r.tableCreate('MeetingMember');
+  } catch (e) {
+    // noop
+  }
+  try {
+    await Promise.all([
+      r.table('MeetingMember').indexCreate('meetingId'),
+      r.table('MeetingMember').indexCreate('teamId'),
+      r.table('MeetingMember').indexCreate('userId')
+    ]);
+  } catch (e) {
+    // noop
+  }
+  try {
+    await r.table('MeetingSettings')
+      .filter({meetingType: RETROSPECTIVE})
+      .update({
+        totalVotes: 5,
```

Comment:
It was _super_ smart of you to do this as Meeting Meetings

Reply:
gotta give the people what they want!